.github/workflows/arm64-build-test.yml

```diff
@@ -15,45 +15,45 @@ on:
 
 env:
   # Common versions
-  GO_VERSION: '1.19'
+  GO_VERSION: '1.22.0'
```

Comment:
You also need to update static check tool version in makefile, you can check my PR for kubevela version bumping: https://github.com/kubevela/kubevela/pull/6560/files#diff-1783d02939953ccc7daacb7a7dd80bbf16c498b08f3fa40a364938938be90a15
Also, please upgrade go version in go.mod as well.

Reply:
damn i missed the go.mod :). Will update and push